packages/datepicker-react/src/DatePicker.tsx

```diff
@@ -138,37 +140,37 @@ export function DatePicker({
     const handleBlur = (e: FocusEvent) => {
         const nextFocusIsInside = componentRef.current && componentRef.current.contains(e.relatedTarget as Node);
         if (onBlur && !nextFocusIsInside) {
-            onBlur(date);
+            onBlur(date, e);
```

Comment:
Kanskje rydde opp inkonsistens mellom funksjonene her, velg enten e eller event, ikke begge.. ikke din feil, men greit å få rydda.

Reply:
Det er nok min feil 😅 Sikkert fint å få rydda når man uansett gjør endringer ja.

Reply:
Takk for forslaget.